src/features/game/expansion/lib/boosts.ts

```diff
@@ -50,6 +50,11 @@ export function isFoodMadeWithHoney(food: Consumable) {
   return cookable ? cookable.ingredients.Honey : false;
 }
 
+export function isFoodMadeWithCheese(food: Consumable) {
+  const cookable = COOKABLES[food.name as CookableName];
+  return cookable ? cookable.ingredients.Cheese : false;
```

Comment:
Suggestion to return a boolean from an "isSomething" fn.

```suggestion
  return !!cookable?.ingredients.Cheese;
```